### PR TITLE
FISH-5976 Trying to access MP Config before Service Locator started during Post boot script

### PR DESCRIPTION
**Pull request by arieki**
## Description

This is a bug fix, to be able to run the `-postbootcommandfile` argument it needs a `TranslatedConfigView` dependency to get and use the `ServiceLocator` and find the necessary config, but it is not at all possible to access the config from within the launcher, as the `TranslatedConfigView` class is inside Felix within OSGi bundle, whereas the one in the launcher is on the system class path.
Therefore creating a service and accessing the `-postbootcommandfile` argument from the StartupContext is necessary

### Blockers

None

### New tests

N/A

### Testing Performed

1. Build the server
2. Create a file *postbootfile* and insert the following command into the file 
```
create-jdbc-connection-pool --restype=java.sql.Driver --driverclassname='oracle.jdbc.xa.client.OracleXADataSource' --property user=${ENV=DB_USER}:password=${MPCONFIG=dbpasswordkey}:URL=${ENV=DB_INSTANCE} mypool
```
3. Start the domain by the following command
```
asadmin start-domain -postbootcommandfile [replace with created postbootfile path]
```
*example:* `asadmin start-domain -postbootcommandfile "C:\postbootfile"`

4. Assert that this error `java.lang.IllegalStateException: Trying to access MP Config before Service Locator started` does not caught

### Testing Environment

Windows 11, OpenJDK 1.8, Maven 3.8.4

## Documentation

N/A

## Notes for Reviewers

N/A

### CODE REVIEW

**Comment by arieki** [comment]
jenkins test please

File: nucleus/core/kernel/src/main/java/com/sun/enterprise/v3/bootstrap/BootCommand.java [hunk 1]
```diff
@@ -37,7 +37,7 @@
  * only if the new code is made subject to such option by the copyright
  * holder.
  */
-package fish.payara.boot.runtime;
+package com.sun.enterprise.v3.bootstrap;
```
Comment (pdudits) [inline nucleus/core/kernel/src/main/java/com/sun/enterprise/v3/bootstrap/BootCommand.java:40]: We can retain payara package here. And as I said earlier, `config-api` is to place to go. It should be safe to add `glassfish-simple-api` dependency there that BootCommand will need.

Comment (arieki) [reply nucleus/core/kernel/src/main/java/com/sun/enterprise/v3/bootstrap/BootCommand.java]: @pdudits  I've pushed to move the `BootCommand` to `config-api`

File: nucleus/core/bootstrap/src/main/java/com/sun/enterprise/glassfish/bootstrap/GlassFishMain.java [hunk 2]
```diff
@@ -135,14 +133,11 @@ public void launch(Properties ctx) throws Exception {
             addShutdownHook();
             gfr = GlassFishRuntime.bootstrap(new BootstrapProperties(ctx), getClass().getClassLoader());
             gf = gfr.newGlassFish(new GlassFishProperties(ctx));
-            doBootCommands(ctx.getProperty("-prebootcommandfile"));
```
[the remark this answers is not in the document]
Comment (arieki) [reply nucleus/core/bootstrap/src/main/java/com/sun/enterprise/glassfish/bootstrap/GlassFishMain.java]: I've put back the `doBootCommands` for `-prebootcommandfile` argument

**Review by pdudits — APPROVED** [review]

**Comment by pdudits** [comment]
Jenkins test please

**Pull request merged**